### PR TITLE
Finished FRAG-18 Task

### PR DESCRIPTION
**Pull request by ItzSveno**

### CODE REVIEW

**Comment by dlanaras** [comment]
Thats a really good job

File: DB/fragile.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE Fraccounts (
+    id integer NOT NULL PRIMARY KEY,
+    name CLOB NOT NULL,
+    password CLOB NOT NULL
+);
```
Comment (laurinneff) [inline DB/fragile.sql:5]: why no ending newline

Comment (dlanaras) [reply DB/fragile.sql:5]: no need fo this still merging

Comment (laurinneff) [reply DB/fragile.sql:5]: nooooooooooooooo

**Pull request merged**